nextage_gazebo/launch/nextage_gazebo_control.launch

```diff
@@ -5,11 +5,11 @@
   <!-- load the controllers -->
   <node name="controller_spawner" pkg="controller_manager"
         type="spawner" respawn="false"
-        output="screen" ns="/NextageOpen" args="joint_state_controller
-                                           L_trajectory_controller
-                                           R_trajectory_controller
-                                           H_trajectory_controller
-                                           C_trajectory_controller
+        output="screen" args="joint_state_controller
```

Comment:
I'm personally a fan of namespacing per robot, for whenever multi-robot thing will become more realistic.